IntegrationWithPharmacies/Controllers/ReportController.cs

```diff
@@ -96,9 +165,36 @@ private DateTime convertStringToDate(String date)
         }
         private SftpConfig getConfig()
         {
-            return new SftpConfig { Host = "192.168.56.1", Port = 22, UserName = "tester", Password = "password" };
+            return new SftpConfig { Host = "192.168.1.244", Port = 22, UserName = "tester", Password = "password" };
         }
-    }
+        public void SendNotificationAboutReport()
+        {
+            try {sendMail();}
+            catch (SmtpException exception) { Console.WriteLine(exception.Message); }
 
+        }
+
+        private static void sendMail()
+        {
+            MailMessage mail = new MailMessage();
+            SmtpClient SmptServer = new SmtpClient("smtp.gmail.com");
+
+            mail.From = new MailAddress("ourhospital9@gmail.com");
+
+            mail.To.Add("pharmacyisa@gmail.com");
+
+            mail.Subject = "Notification about send file";
+
+            mail.Body = "Body of mail address";

```

Comment:
Add a constructor that receives parameters to the MailMessage class to avoid explicitly adding values ​​to parameters.